sidebar.js

```diff
@@ -263,23 +263,26 @@ class TechSidebar extends React.Component {
         if (node.prereqs && node.prereqs.filter(prereq => prereq !== "").length > 0) {
             let prereqElements = [];
             node.prereqs.filter(prereq => prereq !== "").forEach(prereq => {
+                let tech = this.findTechByName(prereq);
```

Comment:
lifted this out to a temp variable so that `filter` in `findTechByName` doesn't have to run several times.